Basic/Basic/OrganizationInfoViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  OrganizationInfoViewController.swift
+//  Basic
+//
+//  Created by Dima Bondar on 16.08.17.
+//  Copyright © 2017 Dima. All rights reserved.
+//
+
+import UIKit
+
+class OrganizationInfoViewController: UIViewController
+{
+
+    var organization:Organization?
+    var salarySum:Int32?
+    
+    override func viewDidLoad()
+    {
+        super.viewDidLoad()        
+    }
+
+    override func didReceiveMemoryWarning()
+    {
+        super.didReceiveMemoryWarning()
+    }
+
+    @IBAction func  showSalarySum(_ sender: UIButton)
+    {
+        if (salarySum == nil)
+        {
+            salarySum = 0;
+            for item:Employee in (organization!.employees)!
+            {
+                salarySum! +=  item.salary
+            }
+        }
+        
+        let alert = UIAlertController(title: "Organization: \(organization!.name!)", message: "Sum of salaries of all employees is: \(salarySum!)", preferredStyle: UIAlertControllerStyle.alert)
```

Comment:
When specifying enum values, you may omit enum's name. So replace `UIAlertControllerStyle.alert` with just `.alert`